README.md

```diff
@@ -145,7 +145,7 @@ version](https://github.com/minio/operator/releases/) of the MinIO
 Kubernetes Plugin:
 
 ```sh
-wget https://github.com/minio/operator/releases/download/v4.2.2/kubectl-minio_4.2.2_linux_amd64 -O kubectl-minio
+wget https://github.com/minio/operator/releases/download/v4.2.7/kubectl-minio_4.2.7_linux_amd64 -O kubectl-minio
```

Comment:
Not sure why we need to document this since krew plugin is published and that is where users should get this binary from? @kanagarajkm @ravindk89

Reply:
@harshavardhana I thought we were moving away from using `krew` to install the Operator Plugin?

If the concern is not having to update our documentation with each release, then we can revert to `krew`. I think its important for us to point out how to download a specific version, since afaik `krew` does not support specifying a single version or version pinning.